docs/keys-config/prd.md

```diff
@@ -52,6 +52,12 @@ type ValidatorKeyStore = {
 	 * An attester address may only appear once across all configuration blocks across all keystore files.
 	 */
 	attester: EthAccounts;
+	/** 
+	 * One or more BLS attester keys to handle in this configuration block.
+	 * These keys map 1 to 1 with the attester accounts above.
+	 * So for example, if a mnemonic is used here it should specify the same number of keys as in 'attester'.
+	 */
+	blsAttester?: BLSAccounts;
```

Comment:
Should we add a section on what happens if this is not set? Is the eth private key used to generate the bls one? Do we throw if the eth config uses remote signer and there is no corresponding bls entry?